test/cluster/mzcompose.py

```diff
@@ -3164,3 +3164,85 @@ def subscribe():
                     ), e
             for thread in threads:
                 assert not thread.is_alive(), f"Thread {thread.name} is still running"
+
+
+def workflow_test_refresh_mv_warmup_after_restart(
+    c: Composition, parser: WorkflowArgumentParser
+) -> None:
+    """
+    Regression test for https://github.com/MaterializeInc/materialize/issues/25279
+    Test that bootstrapping selects an as_of for an MV dataflow in a way that allows it to warm up before its next
+    refresh.
+    """
+
+    with c.override(
+        Materialized(
+            additional_system_parameter_defaults={
+                "enable_refresh_every_mvs": "true",
+            },
+        ),
+        Testdrive(no_reset=True),
+    ):
+        c.down(destroy_volumes=True)
+
+        c.up("materialized")
+        c.up("clusterd1")
```

Comment:
`clusterd1` is not needed anymore.

Reply:
Yeah, I was not sure about this one, but then I forgot to try it without.